edison/src/main/java/edu/illinois/cs/cogcomp/edison/annotators/GazetteerViewGenerator.java

```diff
@@ -47,6 +47,7 @@
  *
  * @author Vivek Srikumar
  */
+@Deprecated
```

Comment:
Maybe add a comment that "instead of this you should use `SimpleGazetteerAnnotator`"?